edx-platform/pearson-learninghub-theme/lms/templates/student_account/register.underscore

```diff
@@ -46,7 +46,7 @@
             </div>
             <div class="section-title lines">
                 <h3>
-                    <span class="text"><%- gettext("or create a new one here") %></span>
+                    <span class="text">Create a new Learning Hub account here</span>
```

Comment:
I agree with @diegomillan We should interpolate the platform name and mark the text for extraction. @luismorenolopera